ProductWarehouse.Persistence.PostgreSQL/Constants/Dapper/CommandConstants.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace ProductWarehouse.Persistence.PostgreSQL.Constants.Dapper;
+public static class CommandConstants
```

Comment:
Maybe name something like that:
CommandConstants.cs -> ProductMutatableQueriesContant.cs 

Or in one file different domain areas:
CommandConstants.cs  -> MutateConstant.cs  
and inside to have ProductMutatableQueriesContant classes per domain areas